src/st/np3/32830.c

```diff
@@ -42,7 +43,7 @@ void func_801B28E4(Entity* self) {
         case 0:
             newEntity = AllocEntity(&g_Entities[224], &g_Entities[256]);
             if (newEntity != NULL) {
-                CreateEntityFromCurrentEntity(E_801B5DE8, newEntity);
+                CreateEntityFromCurrentEntity(E_LIGHTNING_THUNDER, newEntity);
```

Comment:
This was an interesting one - spawning `func_801B28E4` caused the sky to light up, then spawning `E_801B5DE8` caused a thunder noise to play. There's no SFX call here so I'm not 100% sure where that came from. 

Unpausing crashed the game too 😄